test/ejabberd_tests/tests/muc_light_SUITE.erl

```diff
@@ -231,6 +248,60 @@ disco_features(Config) ->
                                                      {attr, <<"var">>}]),
             escalus:assert(is_stanza_from, [?MUCHOST], Stanza)
         end).
+%% The room list is empty
+disco_rooms_empty_page_infinity(Config) ->
```

Comment:
This and `disco_rooms_empty_page_1` looks very similar. I noticed only one different line. Can they be somehow refactored in order not to duplicate code? I think the same applies to next 2 test cases.